server/src/main/java/com/google/musicanalysis/site/OAuthCallbackServlet.java

```diff
@@ -35,6 +38,11 @@
   /** @return A key that is used to store authentication state in a session cookie. */
   protected abstract String getSessionServiceKey();
 
+  /** @return A key that is used to store authentication state in a session cookie. */
+  protected String getSessionTokenKey() {
+    return Constants.YOUTUBE_SESSION_TOKEN_KEY;
+  }
```

Comment:
This should exist in the child class, not here. There should be no references to Youtube in this file.

Reply:
This function should either be abstract or only exist in the Youtube file.